src/Avalonia.Controls/Primitives/SelectingItemsControl.cs

```diff
@@ -1042,7 +1042,7 @@ private void UpdateSelectedItem(int index, bool clear = true)
                 RaiseEvent(e);
             }
 
-            if (AutoScrollToSelectedItem)
+            if (AutoScrollToSelectedItem && _selectedItem != null)
```

Comment:
I think using `_selectedIndex != -1` would be a better idea because there may be null items in the list.